monitor/linuxmonitor/cgnetcls/cgnetcls.go

```
@@ -170,6 +170,12 @@ func (s *netCls) DeleteCgroup(cgroupname string) error {
 	return nil
 }
 
+// GetAssignedMarkVal -- Gets the mark val assigned to the group
+func GetAssignedMarkVal(cgroupName string) string {
+	mark, _ := ioutil.ReadFile(filepath.Join(basePath, TriremeBasePath, cgroupName, markFile))
+	return string(mark[:len(mark)-1])
```

this panic is len(mark) is 0, which can happen as you ignore the errors